apps/nextjs-website/src/components/templates/TutorialTemplate/TutorialTemplate.tsx

```diff
@@ -55,41 +55,66 @@ const TutorialTemplate = ({
           />
         </EContainer>
       )}
-      <EContainer sx={{ paddingY: 7 }}>
-        <FragmentProvider>
-          <Grid container>
-            <Grid item xs={12} lg={9}>
-              <Typography
-                component='h1'
-                sx={{
-                  fontSize: '38px',
-                  fontWeight: 700,
-                  lineHeight: '42px',
-                  paddingY: 2,
-                }}
-              >
-                {title}
-              </Typography>
-              {parts && (
-                <Box mt={1}>
-                  {parts.map((part, index) => (
-                    <PartRenderer key={index} part={part} />
-                  ))}
-                </Box>
-              )}
-            </Grid>
-            <Grid item xs={false} lg={3}>
-              <Box
-                sx={{
-                  position: 'sticky',
-                  top: 200,
-                  maxWidth: '270px',
-                }}
-              ></Box>
-            </Grid>
-          </Grid>
-        </FragmentProvider>
-      </EContainer>
+      <FragmentProvider>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: { xs: 'column', lg: 'row' },
+            maxWidth: '1156px',
+            margin: '0 auto',
+            paddingY: '56px',
+          }}
+        >
+          <Box
+            sx={{
+              flexGrow: { lg: 1 },
+              maxWidth: {
+                xs: '100%',
+                lg: '822px',
+              },
+            }}
+          >
+            <Typography
+              component='h1'
+              sx={{
+                fontSize: '38px',
+                fontWeight: 700,
+                lineHeight: '42px',
+                paddingY: 2,
+              }}
+            >
+              {title}
+            </Typography>
+            {parts && (
+              <Box mt={1}>
+                {parts.map((part, index) => (
+                  <PartRenderer key={index} part={part} />
+                ))}
+              </Box>
+            )}
+          </Box>
+          <Box
+            sx={{
+              display: { xs: 'none', lg: 'initial' },
+              position: 'relative',
+              // 78px is the height of the header, 80px is the height of the product header
+              paddingTop: '158px',
+              paddingLeft: '64px',
+              width: { lg: '270px' },
+            }}
+          >
+            <Box
+              sx={{
+                position: 'sticky',
+                maxWidth: '270px',
+                top: 144,
+              }}
+            >
+              <PartRendererMenu parts={parts ?? []} />
+            </Box>
```

Comment:
The Box is not well positioned 
![image](https://github.com/user-attachments/assets/bf8cd9c9-7d76-41e7-89fb-c9a828620ef9)

the title "IN QUESTA PAGINA" shouldn't be visible if there are no parts with titles
![image](https://github.com/user-attachments/assets/257e0c0e-acf5-4e7e-8730-5fc1543d1c63)

cfr. https://www.figma.com/design/fQkOTQmYbcLVOdZ6NRjVXf/24-Q1-e-Q2---DevPortal?node-id=25-497&t=K3avOge6ijrnSjxx-4